jazz_assets/components/crud/update.js

```diff
@@ -0,0 +1,66 @@
+// =========================================================================
+// Copyright © 2017 T-Mobile USA, Inc.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+/**
+    Update Asset-Catalog by id
+    @module: update.js
+    @description: CRUD functions for asset catalog
+    @author:
+    @version: 1.0
+**/
+
+const utils = require("../utils.js")(); //Import the utils module.
+const _ = require("lodash");
+
+module.exports = (assets_id, update_data, onComplete) => {
+    var docClient = utils.initDocClient();
+    var params = {
+        TableName: global.ASSETS_TABLE,
+        Key: {
+            "id": assets_id
+        }
+    };
+
+    var keys_list = global.global_config.ASSETS_EDITABLE_FIELDS;
+    var update_exp = "";
+    var attributeValues = {};
+    var attributeNames = {};
+    var count = 0;
+    keys_list.map((key) => {
+        var key_name = key;		
+		if(_.includes(_.keys(update_data), key)){
```

Comment:
you can replace lodash with ES5 native function here. You have done this before.